packages/server-wallet/src/wallet/__test__/fixtures/test-channel.ts

```diff
@@ -172,6 +191,46 @@ export class TestChannel {
   static get emptyPayload(): Payload {
     return {walletVersion: WALLET_VERSION};
   }
+
+  public get startBal(): number {
+    return this.startBals.reduce((s, n) => s + n);
+  }
+
+  public async insertInto(store: Store, args: InsertionParams): Promise<DBOpenChannelObjective> {
```

Comment:
Again a docstring would be nice, explaining all the things that happen when this method is called. 
```suggestion
/**
* Calls addSigningKey, pushMessage, updateFunding, ensureObjective and approveObjective on the supplied store.
*/
  public async insertInto(store: Store, args: InsertionParams): Promise<DBOpenChannelObjective> {
```